Disaster Recovery/SafeHaven 4/Register-DataCenters-SRN-and-Peer-SRN.md

```diff
@@ -0,0 +1,128 @@
+
+### Assumptions
+
+SafeHaven Cluster already installed with proper network connectivity between CMS-SRN and SRN-SRN.
+
+### Register Data Centers
+
+#### VMware Data Center
+
+Within the SafeHaven Console, right-click on the **Administrator@Cluster** in the Navigation Tree on the left and select **Register Data Center** from the drop-down menu.
+
+![Upgrade](../../images/SH4.0/Cluster/07.png)
+
+Enter **Data Center Name** and select the **Data Center Type**. Click **Register**.
+
+![Upgrade](../../images/SH4.0/Cluster2/01.png)
+
+Now click on the Data Center you registered and under the **Properties** Panel click on **Register**.
+
+![Upgrade](../../images/SH4.0/Cluster2/03.png)
+
+Enter the **vCenter Identifier** (it can be any name), vCenter **IP Address/Host Name**, **UserName** and **Password** the VMware Data Center administration and Click **Next**.
+
+![Upgrade](../../images/SH4.0/Cluster2/04.png)
+
+**Properties Panel** shows the vCenter Server information.
+
+![Upgrade](../../images/SH4.0/Cluster2/15.jpg)
+
+#### Centurylink Cloud Data Center
+
+Within the SafeHaven Console, right-click on the **Administrator@Cluster** in the Navigation Tree on the left and select **Register Data Center** from the drop-down menu.
+
+![Upgrade](../../images/SH4.0/Cluster/07.png)
+
+Enter **Data Center Name** and select **CenturyLink Cloud** as the **Data Center Type**. Click **Register**.
+
+![Upgrade](../../images/SH4.0/Cluster2/02.png)
+
+Now click on the Data Center you registered and under the **Properties** Panel click on **Change Credentials**.
+
+![Upgrade](../../images/SH4.0/Cluster2/05.png)
+
+Enter your **CenturyLink Portal Username** and **Password** for CenturyLink Cloud administration. Click **Next**.
+
+![Upgrade](../../images/SH4.0/Cluster2/16.jpg)
+
+Select your **parent account** or **sub-account** as well as the CenturyLink Data Center **Location** select **Finish**.
+
+![Upgrade](../../images/SH4.0/Cluster2/17.jpg)
+
+**Properties Panel** shows the CenturyLink Cloud information.
+
+![Upgrade](../../images/SH4.0/Cluster2/18.jpg)
+
+#### Manual Data Center
+
+For Standalone ESXi Hosts or DCC Site type: Within the SafeHaven Console, right-click on the **Administrator@Cluster** in the Navigation Tree on the left and select **Register Data Center** from the drop-down menu.
+
+Enter **Data Center Name** and select **Manual** as the **Data Center Type**. Click **Register**. No credentials required for a Manual site.
+
+![Upgrade](../../images/SH4.0/Cluster2/19.png)
+
+**NOTE**: No credentials required for a Manual site.
+
+**Properties Panel** shows the CenturyLink Cloud information.
+
+![Upgrade](../../images/SH4.0/Cluster2/20.png)
+
+### Register SRN
+
+**Right-click** on the any registered **Data Center** within the Navigation Tree and select **Register SRN** from the drop-down menu.
+
+![Upgrade](../../images/SH4.0/Cluster2/08.png)
+
+Fill in the following fields:
+1)SRN hostname as the **Node Name**

```

Comment:
Change numbering from `1)` to `1.`. Also include a space after the `.`
This will make this KB consistent with the rest of the KBs.